modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -71,6 +71,14 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.5 also includes the following bug fixes:
 
+=== Select would in some situations when it was reused or otherwise reset not have an initial selected value when expected to.
```

Comment:
It might be good to clarify that it's the select element? Not a requirement, but 'select' is a bit broad and can be a bit tricky to understand?

Reply:
```suggestion
=== The selected element would in some situations, when it was reused or otherwise reset not have an initial selected value when expected to.
```

Reply:
@HAFRMO Happy with the above change?